stats/stats_test.go

```diff
@@ -81,6 +83,55 @@ var (
 	}
 	// The id for which the service handler should return error.
 	errorID int32 = 32202
+
+    // Server Stats
```

Comment:
Please fix the documentation everywhere. Remove extra spaces at start and end etc.

Reply:
Everything should be resolved now. Thank you for your help!